src/coreclr/binder/inc/assemblyidentity.hpp

```diff
@@ -30,12 +30,9 @@ namespace BINDER_SPACE
             IDENTITY_FLAG_PUBLIC_KEY_TOKEN       = 0x004,
             IDENTITY_FLAG_PUBLIC_KEY             = 0x008,
             IDENTITY_FLAG_CULTURE                = 0x010,
-            IDENTITY_FLAG_LANGUAGE               = 0x020,
             IDENTITY_FLAG_PROCESSOR_ARCHITECTURE = 0x040,
             IDENTITY_FLAG_RETARGETABLE           = 0x080,
             IDENTITY_FLAG_PUBLIC_KEY_TOKEN_NULL  = 0x100,
-            IDENTITY_FLAG_CUSTOM                 = 0x200,
-            IDENTITY_FLAG_CUSTOM_NULL            = 0x400,
```

Comment:
IDENTITY_FLAG_CUSTOM was to support "custom" attribute that would have a binary blob as a value. We would not use the blob for anything. Managed `AssemblyName` would have no knowledge that `custom` existed.

Modulo validation that "custom" is not duplicated and the blob content is a valid hex, it is not any different from the treatment of unrecognized attributes.